laravel-api/api/User/Models/User.php

```diff
@@ -40,14 +42,16 @@ class User extends Model implements
      * @var array
      */
     protected $fillable = [
-        // 'domain_uuid',
+        'domain_uuid',
```

Comment:
Neither domain uuid, not salt must not be fillable. As well as other fields added here. Let's discuss

Reply:
# Ban field!